packages/eas-cli/src/project/publish.ts

```diff
@@ -245,7 +249,9 @@ export async function filterOutAssetsThatAlreadyExistAsync(
   return missingAssets;
 }
 
-export async function uploadAssetsAsync(assetsForUpdateInfoGroup: CollectedAssets): Promise<void> {
+export async function uploadAssetsAsync(
```

Comment:
Document that this returns the number of uploaded assets, including assets that have already been uploaded.

Reply:
added a type with named properties that should resolve this without resorted to docstrings